ads/_config.js

```diff
@@ -266,6 +266,14 @@ export const adConfig = {
 
   flite: {},
 
+  fluct: {
+    preconnect: [
+      'https://cdn-fluct.sh.adingo.jp',
+      'https://s.sh.adingo.jp',
+      'https://i.adingo.jp',
+    ],
```

Comment:
Please consider implementing the render-start and no-content-available APIs ([see Available APIs](https://github.com/voyagegroup/amphtml/blob/f5336dd4b61b7e80409a6690b120ec6af6d34d36/ads/README.md#available-apis)), which helps AMP to provide user a much better ad loading experience.

Please let me know if there's any questions on it.